staking/availability/measure.go

```diff
@@ -108,12 +108,6 @@ func bumpCount(
 					wrapper.Counters.NumBlocksSigned, common.Big1,
 				)
 			}
-
```

Comment:
Redudant write ?

Reply:
yea, for non-staking txn related validatorWrapper changes, there is no need to do UpdateValidatorWrapper. Just directly change on the live object using state.ValidatorWrapper(addr, false). Because if there is an error, the whole processing of state will error out. No need to revert changes.